gui/src/vmm/hv/linux/mod.rs

```diff
@@ -26,7 +26,7 @@ mod run;
 
 pub fn new(cpu: usize, ram: Ram, debug: bool) -> Result<Kvm, KvmError> {
     // Open KVM device.
-    let kvm = unsafe { open(b"/dev/kvm\0".as_ptr().cast(), O_RDWR) };
+    let kvm = unsafe { open(c"/dev/kvm".as_ptr(), O_RDWR) };
```

Comment:
cbindgen does not support this literal yet. Right now it does not break because this file is not processed by cbindgen due to`cfg` condition.